Sources/TelemetryClient/LogHandler.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import OSLog
```

Comment:
@easydev991 Don't you need to make sure that this can be imported before importing it? I imagine building on systems without access to OSLog would make the code fail.